core/assets/bundles/bundle.properties

```diff
@@ -660,7 +660,6 @@ keybind.press.axis = Press an axis or key...
 keybind.screenshot.name = Map Screenshot
 keybind.move_x.name = Move x
 keybind.move_y.name = Move y
-keybind.mouse_move.name = Follow Mouse
```

Comment:
Removed by mistake ?

Reply:
oops

Reply:
It wasn't removed, it was newly added today and this PR is just behind.

Reply:
@joshuaptfan i just added